### ISSUE TITLE
Replace file() with open() in twisted/spread

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8521]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8521|
|Type|enhancement|
|Created|2016-06-28 06:17:37Z|
|Branch|https://github.com/twisted/twisted/tree/8521-rodrigc-file|

In Python 3, file() is gone (see https://docs.python.org/3/whatsnew/3.0.html#builtins ), and needs to be replaced with open()
which works on Python 2 and 3

<details><summary>Searchable metadata</summary>

```
trac-id__8521 8521
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8521_rodrigc_file 8521-rodrigc-file
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467094657830335 1467094657830335
changetime__1467112522030019 1467112522030019
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa commented|
|-|-|

This references this PR: https://github.com/twisted/twisted/pull/245

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_3"></a>|@Lukasa set owner to @rodrigc|
|-|-|

Builders are all green, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset 9a2ef10ae4e23ca7f739418c52b7e6e45a3436d2
```
#!CommitTicketReference repository="" revision="9a2ef10ae4e23ca7f739418c52b7e6e45a3436d2"
Merge 8521-rodrigc-file: Replace file() with open() in twisted/spread

Author: rodrigc
Reviewer: Lukasa
Fixes: #8521
```
